diff -rupN pmount-0.9.23/Makefile.in pmount-0.9.23-new/Makefile.in
--- pmount-0.9.23/Makefile.in	2010-06-15 23:23:08.000000000 +0000
+++ pmount-0.9.23-new/Makefile.in	2021-05-18 11:02:27.725380146 +0000
@@ -242,7 +242,7 @@ target_alias = @target_alias@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = etc man po src tests
+SUBDIRS = etc man src
 EXTRA_DIST = \
        intltool-extract.in \
        intltool-update.in  \
diff -rupN pmount-0.9.23/configure.ac pmount-0.9.23-new/configure.ac
--- pmount-0.9.23/configure.ac	2010-06-15 23:22:53.000000000 +0000
+++ pmount-0.9.23-new/configure.ac	2021-05-18 11:02:35.215380248 +0000
@@ -5,7 +5,7 @@ AM_CONFIG_HEADER(config.h)
 AC_CONFIG_SRCDIR(src)
 AM_INIT_AUTOMAKE(pmount, 0.9.23)
 
-AC_PROG_INTLTOOL([0.21])
+#AC_PROG_INTLTOOL([0.21])
 
 AM_MAINTAINER_MODE
 AC_ISC_POSIX
